lib/postgrex/pgpass.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Postgrex.Pgpass do
+  @moduledoc """
+  Parses .pgpass file to obtain database credentials
+  """
+
+  @pgpass_path Application.get_env(:postgrex, :pgpass, Path.join(System.user_home, '.pgpass'))
```

Comment:
Sorry I meant we would get rid of this, I was rather interested in the default path being correct for windows.